taskipy/task_runner.py

```diff
@@ -22,6 +25,14 @@ def run(self, task_name: str, args: List[str]) -> int:
                 command, shell=True, cwd=Path(self.__pyproject_path).parent
             )
 
+            def stop_handler(signum: int, frame) -> None:  # pylint: disable=unused-argument
+                # pylint: disable=W0640
+                print(f'Sending signal {signum} to PID: {process.pid}')
+                process.send_signal(signum)
+
+            for sig in CATCHABLE_SIGNALS:
```

Comment:
```suggestion
            for sig in list(signal.Signals):
```
We can use this enum since taskipy only supports python 3.6+

Reply:
after seeing your comment about not being able to catch all signals indiscriminately, let's rename this PR into "propagate SIGTERM". So in this line, we can simply hardcode SIGTERM:
```suggestion
            signal.signal(signal.SIGTERM, send_signal_to_task_process)
```